docs/controls/RemoteDevicePicker.md

```diff
@@ -0,0 +1,68 @@
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, RemoteDevicePicker, picker
+---
+
+# RemoteDevicePicker Control 
+
+The [RemoteDevicePicker](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.controls.remotedevicepicker) gives you a list of Remote Systems. All the systems must be signed in with the same Microsoft Account (MSA)
+
+**Make sure you enable "RemoteSystem" Capability in `package.appxmanifest`**
+
+## Syntax
+
+```c#
+RemoteDevicePicker remoteDevicePicker = new RemoteDevicePicker()
+{
+    Title = "Pick Remote Device",
+    SelectionMode = RemoteDevicesSelectionMode.Multiple
+};
+var result = await remoteDevicePicker.PickDeviceAsync();
+await new MessageDialog($"You picked {result.Count.ToString()} Device(s)" + Environment.NewLine + string.Join(",", result.Select(x => x.DisplayName.ToString()).ToList())).ShowAsync();
+```
+
+You can also use default filer types for initailizing. Like Below.
+
+```c#
+RemoteDevicePicker remoteDevicePicker = new RemoteDevicePicker(RemoteSystemDiscoveryType.Proximal, RemoteSystemAuthorizationKind.Anonymous, RemoteSystemStatusType.Any)
+{
+    Title = "Pick Remote Device",
+	SelectionMode = RemoteDevicesSelectionMode.Multiple
+};
+
+var remoteSystems = await remoteDevicePicker.PickDeviceAsync();
+await new MessageDialog($"You picked {remoteSystems.Count().ToString()} Device(s)" + Environment.NewLine + string.Join(",", remoteSystems.Select(x => x.DisplayName.ToString()).ToList())).ShowAsync();
+```
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| SelectionMode | RemoteDevicesSelectionMode | Gets or sets the DeviceList Selection Mode. Defaults to RemoteDevicesSelectionMode.Single |
+| ShowAdvancedFilters | Boolean | Gets or sets a value indicating whether Advanced Filters visible or not |
+
+## Sample Code
+
+[RemoteDevicePicker Sample Page Source](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/RemoteDevicePicker). You can see this in action in [Windows Community Toolkit Sample App](https://www.microsoft.com/store/apps/9NBLGGH4TLCQ).
+
+## Default Template 
+
+[RemoteDevicePicker XAML File](https://github.com/Microsoft/UWPCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI.Controls/RemoteDevicePicker/RemoteDevicePicker.xaml) is the XAML template used in the toolkit for the default styling.
+
+## Requirements
+
+| Device family | Universal, 10.0.16299.0 or higher |
+| --- | --- |
+| Namespace | Microsoft.Toolkit.Uwp.UI.Controls |
+| NuGet package | [Microsoft.Toolkit.Uwp.UI.Controls](https://www.nuget.org/packages/Microsoft.Toolkit.Uwp.UI.Controls/) |
+
+## API
+
+* [RemoteDevicePicker source code](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI.Controls/RemoteDevicePicker)
+
+## Related Topics
+
+* [Project Rome](https://developer.microsoft.com/en-us/windows/project-rome)
```

Comment:
More links for both topics:
https://docs.microsoft.com/en-us/windows/uwp/launch-resume/connected-apps-and-devices
https://docs.microsoft.com/en-us/windows/uwp/launch-resume/communicate-with-a-remote-app-service
https://github.com/Microsoft/Windows-universal-samples/tree/master/Samples/AppServices